Client/Frontend/Login/LoginInfoViewController.swift

```diff
@@ -0,0 +1,380 @@
+        case delete
+    }
+    
+    // MARK: ItemType
+    
+    enum InfoItem: Int, CaseIterable {
+        case websiteItem
+        case usernameItem
+        case passwordItem
+    }
+    
+    weak var settingsDelegate: SettingsDelegate?
+
+    // MARK: Private
+    
+    private let profile: Profile
+    private weak var websiteField: UITextField?
+    private weak var usernameField: UITextField?
+    private weak var passwordField: UITextField?
+    
+    private var loginEntry: Login {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+    private var isEditingFieldData: Bool = false {
+        didSet {
+            if isEditingFieldData != oldValue {
+                tableView.reloadData()
+            }
+        }
+    }
+    
+    private var formattedCreationDate: String {
+        let date = Date(timeIntervalSince1970: TimeInterval(loginEntry.timeCreated / 1_000_000))
+        let dateFormatter = DateFormatter().then {
+            $0.locale = .current
+            $0.dateFormat = "EEEE, MMM d, yyyy"
+        }
+
+        return dateFormatter.string(from: date)
+    }
+    
+    // MARK: Lifecycle
+
+    init(profile: Profile, loginEntry: Login) {
+        self.loginEntry = loginEntry
+        self.profile = profile
+        super.init()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        navigationItem.do {
+            $0.title = URL(string: loginEntry.hostname)?.baseDomain ?? ""
+            $0.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .edit, target: self, action: #selector(edit))
+        }
+
+        tableView.do {
+            $0.accessibilityIdentifier = "Login Details"
+            $0.register(CenteredButtonCell.self)
+            $0.register(LoginInfoTableViewCell.self)
+            $0.registerHeaderFooter(SettingsTableSectionHeaderFooterView.self)
+            $0.tableFooterView = SettingsTableSectionHeaderFooterView(
+                frame: CGRect(width: tableView.bounds.width, height: 1.0))
+            $0.estimatedRowHeight = 44.0
+            #if swift(>=5.5)
+            if #available(iOS 15.0, *) {
+                $0.sectionHeaderTopPadding = 0
+            }
+            #endif
+        }
+    }
+}
+
+// MARK: TableViewDataSource - TableViewDelegate
+
+extension LoginInfoViewController {
+    
+    override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let headerView = tableView.dequeueReusableHeaderFooter() as SettingsTableSectionHeaderFooterView
+        headerView.titleLabel.text = Strings.Login.loginInfoDetailsHeaderTitle
+        return headerView
+    }
+    
+    override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        switch section {
+            case Section.delete.rawValue:
+                return 1
```

Comment:
why 1 here, not 0?

Reply:
Because of the undesired separator over the Delete Row. This is smooth way of keeping separator over the created row but not underneath it.

Reply:
Can you add a comment in code to explain this?

Reply:
Added https://github.com/brave/brave-ios/pull/4124/commits/a52671a6b3e1b495a7427f86d89b4548203c6804